### ISSUE TITLE
Brak mozliwosci zaslaniania nieprzedstawionych

### ISSUE THREAD

**Issue by apolios** [tjurczyk/arkadia#927]
Brak mozliwosci zaslaniania nieprzedstawionych poprzez klik w oknie kondycji. Nieprzedstawiony jest nieklikalny. Aliasem mozna.

**Comment by Delwing** [comment]
Czy nie możesz zasłonić kiedy łamie Ci linię?

**Comment by apolios** [comment]
tzn?

**Comment by Delwing** [comment]
To może inaczej:

```/cset! scripts.ui.states_window_no_wrap=true``` 
Wypróbuj takie ustawienie i daj znać czy dalej problem występuje.

**Comment by apolios** [comment]
tak, to pomaga, bez tego tez jest dobrze, jesli szort postaci nie zawija sie do nowej linii.

**Comment by Delwing** [comment]
To niestety tak będzie, nic z tym nie poradzimy.
